utils/file/file.go

```diff
@@ -106,7 +111,12 @@ func GetYAML(taskInfo *mesos.TaskInfo) []string {
 // In case they have different depth of folders to keep compose files, GetDirFilesRecv help to get the complete path of
 // compose file
 func GetDirFilesRecv(dir string, files *[]string) {
-	if d, _ := os.Stat(dir); !d.IsDir() && (strings.HasSuffix(dir, ".yml") || strings.HasSuffix(dir, ".yaml") || dir == "yaml") {
+	d, _ := os.Stat(dir)
+	if d == nil {
```

Comment:
Nit: in this case there is no difference but normally as a rule I would not ignore the returned error value. In this specific case `err != nil` implies `d == nil`. How about checking `err != nil` instead of `d == nil`?

Reply:
Thanks for raising this, I will go over the code, and fix all these in next pr.